server/ott-config.ts

```diff
@@ -513,6 +519,11 @@ function postProcessConfig(): void {
 	if (conf.get("mail.enabled")) {
 		validateMail();
```

Comment:
what does mail mean in this context?

Reply:
email. used for password recovery emails.